modules/serverless-install-kafka-odc.adoc

```diff
@@ -54,25 +56,24 @@ spec:
             replicationFactor: <replication_factor> <7>
     sink:
         enabled: true <8>
+    logging:
+        level: INFO <9>
 ----
 <1> Enables developers to use the `KafkaChannel` channel type in the cluster.
 <2> A comma-separated list of bootstrap servers from your AMQ Streams cluster.
 <3> Enables developers to use the `KafkaSource` event source type in the cluster.
 <4> Enables developers to use the Knative broker implementation for Apache Kafka in the cluster.
 <5> A comma-separated list of bootstrap servers from your Red Hat AMQ Streams cluster.
 <6> Defines the number of partitions of the Kafka topics, backed by the `Broker` objects. The default is `10`.
-<7> Defines the replication factor of the Kafka topics, backed by the `Broker` objects. The default is `3`.
+<7> Defines the replication factor of the Kafka topics, backed by the `Broker` objects. The default is `3`. The `replicationFactor` value must be less than or equal to the number of nodes of your Red Hat AMQ Streams cluster.
 <8> Enables developers to use a Kafka sink in the cluster.
+<9> Defines the log level of the Kafka data plane. Allowed values are `TRACE`, `DEBUG`, `INFO`, `WARN` and `ERROR`. The default value is `INFO`.
 +
-[NOTE]
+[WARNING]
 ====
-The `replicationFactor` value must be less than or equal to the number of nodes of your Red Hat AMQ Streams cluster.
+Do not use `DEBUG` or `TRACE` as the logging level in production environments. The outputs from these logging levels are verbose and can degrade performance.
 ====
 
-.. Using the form is recommended for simpler configurations that do not require full control of *KnativeKafka* object creation.
-
-.. Editing the YAML is recommended for more complex configurations that require full control of *KnativeKafka* object creation. You can access the YAML by clicking the *Edit YAML* link in the top right of the *Create Knative Kafka* page.
```

Comment:
not sure if these restructuring things are directly related to the PR.  But the rest seems OK

Reply:
Sort of - it makes more sense for them to be an unordered list IMO because they're options, not sequential steps, and it makes sense I think for the YAML example to be under the YAML step rather than before both the steps. I can change it back if you don't think it makes sense to do this though.